projects/1.16.1/assets/atmospheric/atmospheric/lang/zh_cn.json

```
@@ -0,0 +1,430 @@
+	"block.atmospheric.smooth_arid_sandstone_vertical_slab": "平滑干燥砂岩竖直台阶",
+	"block.atmospheric.arid_sandstone_brick_vertical_slab": "干燥砂岩砖竖直台阶",
+	"block.atmospheric.arid_sandstone_bricks": "干燥砂岩砖",
+	"block.atmospheric.arid_sandstone_brick_stairs": "干燥砂岩砖楼梯",
+	"block.atmospheric.arid_sandstone_brick_slab": "干燥砂岩砖台阶",
+	"block.atmospheric.arid_sandstone_brick_wall": "干燥砂岩砖墙",	
+	
+	"block.atmospheric.red_arid_sand": "干燥红沙",
+	"block.atmospheric.red_arid_sandstone": "干燥红砂岩",
+	"block.atmospheric.red_arid_sandstone_stairs": "干燥红砂岩楼梯",
+	"block.atmospheric.red_arid_sandstone_slab": "干燥红砂岩台阶",
+	"block.atmospheric.red_arid_sandstone_wall": "干燥红砂岩墙",
+	"block.atmospheric.chiseled_red_arid_sandstone": "錾制干燥红砂岩",
+	"block.atmospheric.cut_red_arid_sandstone": "切制干燥红砂岩",
+	"block.atmospheric.cut_red_arid_sandstone_slab": "切制干燥红砂岩台阶",
+	"block.atmospheric.smooth_red_arid_sandstone": "平滑干燥红砂岩",
+	"block.atmospheric.smooth_red_arid_sandstone_stairs": "平滑干燥红砂岩楼梯",
+	"block.atmospheric.smooth_red_arid_sandstone_slab": "平滑干燥红砂岩台阶",
+	
+	"block.atmospheric.red_arid_sandstone_vertical_slab": "干燥红砂岩竖直台阶",
+	"block.atmospheric.cut_red_arid_sandstone_vertical_slab": "切制干燥红砂岩竖直台阶",
+	"block.atmospheric.smooth_red_arid_sandstone_vertical_slab": "平滑干燥红砂岩竖直台阶",
+	"block.atmospheric.red_arid_sandstone_brick_vertical_slab": "干燥红砂岩砖竖直台阶",
+	"block.atmospheric.red_arid_sandstone_bricks": "干燥红砂岩砖",
+	"block.atmospheric.red_arid_sandstone_brick_stairs": "干燥红砂岩砖楼梯",
+	"block.atmospheric.red_arid_sandstone_brick_slab": "干燥红砂岩砖台阶",
+	"block.atmospheric.red_arid_sandstone_brick_wall": "干燥红砂岩砖墙",	
+	
+	"item.atmospheric.yucca_boat": "丝兰木船",
+	"item.atmospheric.yucca_fruit": "丝兰果",
+	"item.atmospheric.roasted_yucca_fruit": "烤丝兰果",
+	"item.atmospheric.yucca_juice": "丝兰果汁",
+	
+	"item.atmospheric.aloe_kernels": "芦荟仁",
+	"item.atmospheric.aloe_leaves": "芦荟叶",
+	"item.atmospheric.aloe_gel_bottle": "芦荟凝胶瓶",
+	
+	"effect.atmospheric.persistence": "持久",
+	"effect.atmospheric.gelled": "胶凝",
+	
+	"biome.atmospheric.dunes": "沙丘",
+	"biome.atmospheric.dunes_hills": "沙丘丘陵",
+	"biome.atmospheric.flourishing_dunes":"繁茂沙丘",
+	
+	"biome.atmospheric.rocky_dunes": "岩石沙丘",
+	"biome.atmospheric.rocky_dunes_hills": "岩石沙丘丘陵",
+	"biome.atmospheric.petrified_dunes": "石化沙丘",
+	
+	"death.attack.yuccaSapling": "%1$s被丝兰树苗切成了片",
+	"death.attack.yuccaSapling.player": "%1$s在试图逃离%2$s时被丝兰树苗切成了片",
+	"death.attack.yuccaFlower": "%1$s被丝兰花切成了片",
+	"death.attack.yuccaFlower.player": "%1$s在试图逃离%2$s时被丝兰花切成了片",
+	"death.attack.yuccaBranch": "%1$s被丝兰树枝切成了片",
+	"death.attack.yuccaBranch.player": "%1$s在试图逃离%2$s时被丝兰树枝切成了片",
+	"death.attack.yuccaLeaves": "%1$s被丝兰树叶切成了片",
+	"death.attack.yuccaLeaves.player": "%1$s在试图逃离%2$s时被丝兰树叶切成了片",
+	"death.attack.barrelCactus": "%1$s撞上了桶仙人掌",
+	"death.attack.barrelCactus.player": "%1$s在试图逃离%2$s时撞上了桶仙人掌",
+	"death.attack.aloeLeaves": "%1$s被芦荟叶戳死了",
+	"death.attack.aloeLeaves.player": "%1$s在试图逃离%2$s时被芦荟叶戳死了",
+	
+	"advancements.husbandry.finish_gateau.title": "何不食……蛋糕？",

```

|Feedback Priority|
|---|
|⚪️ Pebble / Non-blocking but requires future action|

```suggestion
	"advancements.husbandry.finish_gateau.title": "何不食...蛋糕？",
```

---

已修正